frontends/concrete-python/concrete/fhe/compilation/module_compiler.py

```diff
@@ -638,6 +638,7 @@ def compile(
         inputsets: Optional[Dict[str, Union[Iterable[Any], Iterable[Tuple[Any, ...]]]]] = None,
         configuration: Optional[Configuration] = None,
         module_artifacts: Optional[ModuleDebugArtifacts] = None,
+        auto_schedule_run: bool = False,
```

Comment:
This should be a configuration option.

Reply:
it is a module configuration, you want it to work also outside modules ?

Reply:
There are more options like that already. I think it's better in terms of testing (e.g., you can set it as the default config for tests). Also, it'd be localized along with all other options, thus it'd be easier to find.

Reply:
Done.